content/sensu-go/5.21/reference/assets.md

```diff
@@ -93,25 +93,62 @@ Follow the steps outlined in [Contributing Assets for Existing Ruby Sensu Plugin
 For further examples of Sensu users who have added the ability to use a community plugin as an asset, see [this Discourse post](https://discourse.sensu.io/t/how-to-use-the-sensu-plugins-kubernetes-plugin/1286).
 {{% /notice %}}
 
-### Asset path
+### Default cache directory
+
+system  | sensu-backend                         | sensu-agent
+--------|---------------------------------------|-------------
+Linux   | `/var/cache/sensu/sensu-backend`      | `/var/cache/sensu/sensu-agent`
+Windows | N/A                                   | `C:\ProgramData\sensu\cache\sensu-agent`
+
+If the requested asset is not in the local cache, it is downloaded from the asset URL.
+The Sensu backend does not currently provide any storage for assets.
+Sensu expects assets to be retrieved over HTTP or HTTPS.
+
+### Example asset structure
+
+{{< code shell >}}
+sensu-example-handler_1.0.0_linux_amd64
+├── CHANGELOG.md
+├── LICENSE
+├── README.md
+└── bin
+  └── my-check.sh
+└── lib
+└── include
+{{< /code >}}
+
+## Asset path
 
 When you download and install an asset, the asset's local path on disk is exposed to asset consumers via environment variables.
-This allows you to retrieve the asset's path from a check, hook, handler, or mutator either as an environment variable or from a check or hook using a custom function for token substitution, `assetPath`.
+This allows you to retrieve the asset's path as an [environment variable][14] from a check, handler, hook, or mutator or using a [custom function for token substitution][17], `assetPath`, from a check or hook.
 
-For example, if you included a configuration file in the `include` directory of the asset [`sensu-plugins-windows`][4], you can reference the asset from your check in either of these ways:
+### Environment variable for asset path
 
-- `$SENSU_PLUGINS_WINDOWS/include/config.yaml`
-- `${{assetPath "sensu-plugins-windows"}}/include/config.yaml`
+If you included a configuration file in the `include` directory of the asset [`sensu-plugins-windows`][4], you can use the asset path environment variable to reference the asset from your check, handler, hook, or mutator:
+
+`$SENSU_PLUGINS_WINDOWS_PATH/include/config.yaml`
 
 {{% notice note %}}
-**NOTE**: The `assetPath` function is only available where token substitution is available: checks and hooks.
+**NOTE**: Sensu always appends the suffix `_PATH` for environment variables that are automatically generated for asset paths.
 {{% /notice %}}
 
+### Token substitution for asset path
+
 The asset path includes the asset's checksum, which changes every time the asset is updated.
```

Comment:
Most of these words belongs in "why you want this" section. Other salient points we need to cover:

* Most of the time, you won't need to know the path to the asset files on disk
* Sometimes you need this, like to provide an argument to a command.
* Both the environment variable and the token substitution help you get this information. 
* You can use the environment variable in every context, but the token substitution only works in check or hook commands.